src/controllers/mainController.ts

```diff
@@ -1178,6 +1187,19 @@ export default class MainController implements vscode.Disposable {
 
 			await this.sanitizeConnectionProfiles();
 
+			if (e.affectsConfiguration(Constants.cmdObjectExplorerGroupBySchemaFlagName)) {
+				this._objectExplorerProvider.rootNodeConnections.forEach(async (conn) => {
+					if (this.connectionManager.isActiveConnection(conn)) {
+						const rootNodes = await this._objectExplorerProvider.getChildren();
```

Comment:
You're already looping over rootNodeConnections - do you really have to call getChildren each time? It seems redundant to have to do that again for each node.